FCP-INDI/C-PAC
# Monkey improve

## Fixes


Fixes #1384 by @HechengJin0 
Fixes #1381 by @HechengJin0 

## Description

1. Apply inverse coreg matrix for the tissue mask conversions to func space for nuisance.
2. fix bug - ‘Selected Functional Volume’ as functional to anat registration registration EPI input
The image should be skullstripped, should be lined up with the EPI comes from [’mean functional`]
3. change the config field `n4_correct_mean_EPI` to `n4_correct_func_reg_input` to make N4 bias correction work for either func_reg_input is 'Mean Functional' or func_reg_input is 'Selected Functional Volume'

## Tests

1. check tissue masks (especially csf) in nuisance regressor dir. 
2. check functional-to-anatomical registration EPI input
/data3/cnl/monkey_benchmark/cpac_run/monkey_1020_oxford_cpac172dev_monkey_improve_branch_tingmask_benchmark_config_2

## Screenshots

![image](https://user-images.githubusercontent.com/47699110/96755595-eace0f00-13a0-11eb-9c21-4e56baea5e72.png)

## Checklist

  index.md`).
  visible errors.

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
